tools/nodeset_compiler/backend_open62541.py

```diff
@@ -134,8 +135,19 @@ def writeh(line):
 
     def writec(line):
         print(unicode(line), end='\n', file=outfilec)
+        
+    def parentNodeString(node,L):
```

Comment:
This can be simplified

```python
    def parentNodeString(node,L):
        if node is None or node.parent is None:
            return ""
        return str(parentNodeString(node.parent,L)) + "_" + str(node.displayName).upper()
```